src/devices/bus/coco/coco_multi.cpp

```diff
@@ -233,7 +262,19 @@ void coco_multipak_device::device_start()
 
 void coco_multipak_device::device_reset()
 {
-	m_select = 0xFF;
+	m_select = MULTI_SLOT_LOOKUP[ioport("MPI Slot Switch")->read()];
```

Comment:
Is this really adequate?  Wouldn't you have to go through `set_select` to make sure everything's mapped properly?